src/main/resources/db/migration/V1.11__Add_PointsTransaction_table_and_fill_it.sql

```diff
@@ -0,0 +1,18 @@
+create table points_transaction (
+  transaction_id bigint not null auto_increment,
+  reference_id bigint not null,
+  date_creation datetime not null,
+  user_from bigint not null,
+  user_to bigint not null,
+  points int not null ,
+  primary key (transaction_id)
+);
+
+  alter table points_transaction add constraint POINTS_TRANSACTION_TRANSACTION_FK foreign key (reference_id) references points_transaction (transaction_id);
```

Comment:
Please remove the POINTS_TRANSACTION_TRANSACTION_FK constraint. We plan to reference course_ownership or deck_ownership (or any other in the future).

Deck_Ownership_Transaction_FK and Course_Ownership_Transaction_FK will have to be removed also. @OlehPaliuh, I realized it just now.